pkgs/applications/misc/confy/default.nix

```diff
@@ -18,9 +18,9 @@ stdenv.mkDerivation rec {
   pname = "confy";
   version = "0.7.0";
 
-  src = fetchurl {
+  src = fetchzip {
     url = "https://git.sr.ht/~fabrixxm/confy/archive/${version}.tar.gz";
```

Comment:
We have "native" support for `fetchFromSourcehut`...

Reply:
:sweat_smile: I didn't even recognize that as a sourcehut website. This might be important:

https://github.com/NixOS/nixpkgs/blob/611bd732470e3efd42751c35ea1f96098f03192a/pkgs/build-support/fetchsourcehut/default.nix#L43-L45

Reply:
But that's not relevant, as it's a git version control. I'll still open a PR.

Reply:
https://github.com/NixOS/nixpkgs/pull/246730